pkg/v1/cli/discovery/interface.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package discovery is implements discovery interface for plugin discovery
```

Comment:
**nitpick**: I think it'd be worth including some additional context here — perhaps you can copy a paragraph or two straight from your design doc.